core/src/main/scala/kafka/raft/KafkaMetadataLog.scala

```diff
@@ -698,9 +698,7 @@ object KafkaMetadataLog extends Logging {
 
   private def deleteSnapshotFiles(
     logDir: Path,
-    expiredSnapshots: mutable.TreeMap[OffsetAndEpoch, Option[FileRawSnapshotReader]],
-    logging: Logging
-  ): Unit = {
+    expiredSnapshots: mutable.TreeMap[OffsetAndEpoch, Option[FileRawSnapshotReader]]): Unit = {
     expiredSnapshots.foreach { case (snapshotId, snapshotReader) =>
       snapshotReader.foreach { reader =>
         Utils.closeQuietly(reader, "reader")
```

Comment:
The second argument is for logging purposes.
Should we use a more meaningful name instead of "reader"? 
To me, "reader" doesn't provide too much value for troubleshooting.
For example: "FileRawSnapshotReader"

cc @chia7712

Reply:
sounds good :)

Reply:
Hi @shmily7829 
Could you please change "reader" to "FileRawSnapshotReader" ?
For example:
```suggestion
        Utils.closeQuietly(reader, "FileRawSnapshotReader")
```